--- src/components/layout/AuthGuard.tsx
+++ src/components/layout/AuthGuard.tsx
@@ -10,13 +10,13 @@
 
   useEffect(() => {
     if (isReady) {
       const isLoginPage = pathname.includes("/login");
       
       if (!user && !isLoginPage) {
-        router.push("/login");
+        router.push("/presensi-siswa/login");
       }
       if (user && isLoginPage) {
-        router.push("/");
+        router.push("/presensi-siswa/");
       }
     }
   }, [user, isReady, pathname, router]);
